openstack: Update affinity docs (#5274)
openshift/installer: docs/user/openstack/README.md

#### Using a Server Group

> The `serverGroupID` property of the MachineSet will be used to create new
> machines in that OpenStack Server group. The server group must exist in
> OpenStack before the new MachineSet is applied.
> 

Reviewer: ```suggestion
The `serverGroupID` property of the [machine set | `MachineSet` resource] is used to create machines in that OpenStack server group. The server group must exist in OpenStack before you can apply the new [machine set | `MachineSet` resource].
```

Since I'm looking. :)
